client/js/components/statement/listOriginalStatements/ListOriginalStatements.vue

```diff
@@ -0,0 +1,565 @@
+<template>
+  <div>
+    <dp-map-modal
+      ref="mapModal"
+      :procedure-id="procedureId" />
+
+    <dp-loading
+      v-if="isLoading"
+      class="u-mt"/>
+
+    <template v-else>
+      <dp-pager
+        v-if="pagination.currentPage && originalStatements.length > 0"
+        :class="{ 'invisible': isLoading }"
+        class="mt-4 mb-2"
+        :current-page="pagination.currentPage"
+        :total-pages="pagination.totalPages"
+        :total-items="pagination.total"
+        :per-page="pagination.perPage"
+        :limits="pagination.limits"
+        @page-change="fetchOriginalStatementsByPage"
+        @size-change="handleSizeChange"
+        :key="`pager1_${pagination.currentPage}_${pagination.perPage}`" />
+
+      <dp-data-table
+        v-if="originalStatements.length > 0"
+        has-flyout
+        :header-fields="headerFields"
+        is-expandable
+        :items="originalStatements"
+        track-by="id">
+        <template v-slot:externId="{ externId }">
+          <span
+            class="font-semibold"
+            v-text="externId" />
+        </template>
+        <template
+          v-slot:submitter="{ id }">
+          <div v-cleanhtml="getSubmitterName(id)" />
+        </template>
+        <template v-slot:submitDate="{ submitDate }">
+        <span>
+          {{ formatDate(submitDate) }}
+        </span>
+        </template>
+        <template v-slot:shortText="{ shortText }">
+          <div
+            class="line-clamp-3 c-styled-html"
+            v-cleanhtml="shortText" />
+        </template>
+        <template v-slot:phase="{ phase }">
+        <span
+          v-if="phase">
+          {{ phase }}
+        </span>
+        </template>
+        <template
+          v-if="hasPermission('area_statement_anonymize')"
+          v-slot:flyout="{ externId, id }">
+          <dp-flyout>
+            <a
+              class="u-pt-0"
+              :href="Routing.generate('DemosPlan_statement_anonymize_view', { procedureId: procedureId, statementId: id })">
+              {{ Translator.trans('statement.anonymize', { externId: externId }) }}
+            </a>
+          </dp-flyout>
+        </template>
+        <template v-slot:expandedContent="{
+          authorName,
+          fullText,
+          id,
+          isSubmittedByCitizen,
+          polygon,
+          shortText,
+          textIsTruncated
+        }">
+          <div class="u-pt-0_5">
+
+            <!-- Meta data -->
+            <div>
+              <!-- Submission meta data -->
+              <dl class="inline-grid grid-cols-[25%_75%] w-[49%]">
```

Comment:
Just curious: why does this need to be set to 49% width? i suppose the two blocks have that whitespace thingy between them... in this case we could try setting `flex` on the outer div?

besides, i like the grid solution. i think using arbitrary values here is perfectly ok.

Reply:
I think the whitespace thingy is probably the reason for having to reduce the width slightly, but I could not get it fixed, and I thought maybe we can live with a width of 49%. I'm also not sure that using flex would be a good solution here, since the outer `div` contains also the attachments `dl` and the statement text, which we always want to display in their own rows.